config/settings.yml

```diff
@@ -392,8 +392,22 @@ mediated_locations:
     - LOCKED-STK
   SAL3:
     - PAGE-MP
+
+aeon_locations:
   RUMSEYMAP: "*"
   SPEC-COLL: "*"
+  HOOVER: "*"
+  SAL:
```

Comment:
> Currently enabled library/location combinations, and requests need to go to Aeon
> library: SAL, location: L-PAGE-EA, Item type = NH-INHOUSE
> library: SAL3, location: PAGE-AS, Item type = NH-INHOUSE
> library: SAL3, location: PAGE-EA, Item type = LCKSTK
> Note SAL3 PAGE-EA items that do NOT have item type = LCKSTK should display request links, but the request should go to Symphony.

Don't we also need to differentiate by item type, or is that done elsewhere?

Reply:
Hmm. Do we need to borrow the whole `LocationRule` idea from requests, then? idk if Searchworks can differentiate by item type right now

Reply:
Is the best way to do it to pull it out of `building_location_facet_ssim`, or is there a different accessor? I see some "item_type" related stuff but it's mostly in configs.

Reply:
I don't think we're using (or returning) `building_location_facet_ssim`. Your best bet is probably pulling the item types off the `Holdings::Item`: https://github.com/sul-dlss/SearchWorks/blob/master/lib/holdings/item.rb#L60

I did confirm with Sarah much earlier that Aeon-eligible material all have Aeon-eligible types, so at least there's that.

Reply:
ok, took a stab at this using a config syntax similar to what we have over in requests.